Updating pybindgen version to fix 2 memory leaks

lowaa opened the pull request:
Updating pybindgen version, adding a very simple memory test and updating the version

lowaa committed:
Updating pybindgen version to fix 2 memory leaks

DanielLeone commented on requirements.txt:
```diff
@@ -1,6 +1,7 @@
-python-dateutil==2.7.5
\ No newline at end of file
+python-dateutil==2.7.5
+guppy==0.11.1
```
```suggestion
guppy==0.1.11
```

DanielLeone reviewed (approved):
running the memory tests and nothing seems out place :) nice job 👍

lowaa committed:
updating package version

lowaa committed:
updating package version

smills2929 reviewed (approved):
Straightforward PR, I like that there are memory tests. Daniel's suggested code change for guppy version is a good catch.

lowaa committed:
Adding another mem test that leaks
Adding odbcinst for FreeTDS driver
Adding test_mem.sh that runs mem test in a docker container
